packages/eslint-plugin/lib/rules/require-media-dimensions.js

```diff
@@ -0,0 +1,70 @@
+const {elementType, hasProp} = require('jsx-ast-utils');
+
+const {docsUrl, polarisComponentFromJSX} = require('../utilities');
+
+const POLARIS_MEDIA_COMPONENTS = ['Image'];
```

Comment:
Thoughts on embedding this in the library itself as opposed to specifying it via the `mediaElements` option on the consumer-end?

One advantage of the current approach is that we're able to analyze the import specifier and look at the source to ensure that it's actually from `@shopify/polaris` and not an `Image` component defined somewhere else which I imagine could cause problems for consumers.